common/src/java/com/zimbra/common/localconfig/DebugConfig.java

```diff
@@ -230,6 +230,12 @@ private DebugConfig() {
             "defang_style_unwanted_import",
             "@import(\\s)*((\'|\")?(\\s)*(http://|https://)?([^\\s;]*)(\\s)*(\'|\")?(\\s)*;?)");
 
+    public static final String xhtmlWhitelistedTags = value("defang_xhtml_whitelisted_tags",
+        "a,abbr,acronym,blockquote,div,font,h1,h2,h3,h4,h5,h6,img,li,ol,p,span,table,td,th,tr,ul");
+
+    public static final String xhtmlWhitelistedAttributes = value("defang_xhtml_whitelisted_attributes",
+        "abbr,align,alt,border,cellpadding,cellspacing,cite,class,color,colspan,height,href,id,,name,rel,rev,rowspan,size,src,style,title,target,valign,width");
```

Comment:
Does the extra comma between `id` and `name` affect anything?

Reply:
No, it shouldn't but I removed it.